### PR TITLE
Packery setup

### PR DESCRIPTION
**Pull request by gduck**
Have a nice starting page, with user input entering into a 'feature article', and divs resizing on longer input.

### CODE REVIEW

**Comment by h8rry** [comment]
Shouldn't version control `.svg` files with so many code additions

**Comment by gduck** [comment]
ok. add .svg to .gitignore?

File: app/assets/fonts/Canterbury/Canterbury.svg [hunk 1]
```diff
@@ -0,0 +1,635 @@
+<?xml version="1.0" standalone="no"?>
```
Comment (h8rry) [inline app/assets/fonts/Canterbury/Canterbury.svg]: Please put all the font files in Amazon AWS S3. Doing version control for large static assets will make git look very messy.

Comment (gduck) [inline app/assets/fonts/Canterbury/Canterbury.svg]: I don't know how to go about doing this. I assume keep .svg as is locally, but do some sort of special thing with heroku to put fonts on S3...?

Comment (h8rry) [inline app/assets/fonts/Canterbury/Canterbury.svg]: Like how we uploaded images to AWS S3 and linked them to our app, we can do the same for fonts. So the fonts files can be accessed both locally and on production.

**Comment by h8rry** [comment]
@gduck 
Configure Amazon S3 for Cross Origin Resourse Sharing to Host a Web Font:
http://www.html5hacks.com/blog/2012/11/18/configure-amazon-s3-for-cross-origin-resourse-sharing-to-host-a-web-font/

**Comment by h8rry** [comment]
@gduck 
Use transmit to access AWS S3 easily:
- http://panic.com/transmit/
- http://ryansechrest.com/2013/11/accessing-amazon-s3-buckets-via-transmit/

**Comment by h8rry** [comment]
@gduck 

### Steps to host font files on AWS S3:
1. Go to Amazon AWS
2. Go to S3 (similar to dropbox)
3. Create a new "bucket" on S3 (similar to a folder)
4. Upload your font files into the bucket (similar to uploading to dropbox)
5. Find out the exact URL to the files
6. Allow public access to the files
7. Include your font files inside your CSS / HTML
8. Example:

``` css
@font-face {
  font-family: BebasNeue;
  src: url('https://s3.amazonaws.com/html5hacks/BebasNeue.otf');
}
```

**Comment by h8rry** [comment]
@gduck 
In Edit bucket policy, you have to replace 'nvest' with your own bucket name

``` json
{
    "Version": "2008-10-17",
    "Statement": [
        {
            "Sid": "AllowPublicRead",
            "Effect": "Allow",
            "Principal": {
                "AWS": "*"
            },
            "Action": "s3:GetObject",
            "Resource": "arn:aws:s3:::nvest/*"
        }
    ]
}
```

In CORS Configuration Editor

``` html
<CORSConfiguration>
    <CORSRule>
        <AllowedOrigin>*</AllowedOrigin>
        <AllowedMethod>GET</AllowedMethod>
        <MaxAgeSeconds>3000</MaxAgeSeconds>
        <AllowedHeader>Authorization</AllowedHeader>
    </CORSRule>
</CORSConfiguration>

```

**Comment by h8rry** [comment]
screen shot 2015-02-18 at 3 34 41 pm

Looks great

**Pull request merged**